plugins/sudoers/iolog_plugin.h

```diff
@@ -86,6 +86,8 @@ enum client_state {
 /* Remote connection closure, non-zero fields must come first. */
 struct client_closure {
     int sock;
+    struct sudoers_string *host;
+    char ipaddr[INET6_ADDRSTRLEN];
```

Comment:
INET6_ADDRSTRLEN won't be present on non-IPv6 systems

Reply:
fixed